src/index.jsx

```diff
@@ -98,8 +43,6 @@ subscribe(APP_READY, () => {
 });
 
 initialize({
-  messages: [
-    footerMessages,
-  ],
+  messages: [],
```

Comment:
What is this? Is it still necessary?

Reply:
`initialize` fails without `messages` arg. But looking into it, turns out we actually should keep them in. Reverting.